inc/classes/customizer/class-social.php

```diff
@@ -131,6 +131,33 @@ public function setup_fields() {
 			)
 		);
 
+		$control_id = 'wmf_linkedin_label';
+		$this->customize->add_setting(
+			$control_id,
+			array(
+				'default' => __( 'LinkedIn', 'shiro-admin' ),
+			)
+		);
+		$this->customize->add_control(
+			$control_id,
+			array(
+				'label'   => __( 'LinkedIn Label', 'shiro-admin' ),
+				'section' => $section_id,
+				'type'    => 'text',
+			)
+		);
+
+		$control_id = 'wmf_linkedin_url';
+		$this->customize->add_setting( $control_id );
+		$this->customize->add_control(
```

Comment:
Per WIKI-1038 we'll be sunsetting customizer usage eventually — what's the best block-oriented way to manage these options? (Does not need an answer now)